src/app/shared/links/links-list/links-accordeon.component.html

```diff
@@ -0,0 +1,42 @@
+<ng-container *ngIf="{linkTypes: linkTypes$ | async, selectedLinkType: selectedLinkType$ | async} as data">
+  <ng-container *ngIf="data.linkTypes.length > 0; else noLinkTypes">
+  <accordion>
+    <ng-container *ngFor="let linkType of data?.linkTypes; let idx = index;">
+      <accordion-group panelClass="lmr-accordion" [isOpen]="linkType.id === data?.selectedLinkType?.id" (isOpenChange)="isOpenChanged(idx)">
+        <div class="header-wrapper" accordion-heading *ngIf="{iconColors: linkType | asAttributeResource | resourceIconsColors} as iconData">
+          <div class="mr-2">
+            <i *ngIf="(openedGroups$ | async)[idx]" class="fas fa-caret-down"></i>
+            <i *ngIf="!((openedGroups$ | async)[idx])" class="fas fa-caret-right"></i>
+          </div>
+          <icons-presenter class="d-block mr-1"
+                           [colors]="iconData.iconColors.colors"
+                           [icons]="iconData.iconColors.icons">
+          </icons-presenter>
+          <div class="font-weight-bold text-truncate" [title]="linkType.name">{{linkType.name}}</div>
+          <div class="small text-muted">{{document | linksCount:linkType | async}}</div>
+        </div>
+        <div>
+          <links-list-table
+            class="d-block mt-2"
+            [linkType]="data.selectedLinkType"
+            [document]="document"
+            [collection]="otherCollection$ | async"
+            [permissions]="permissions$ | async"
+            [query]="query$ | async"
```

Comment:
I try that too, however, there always isn't the possibility where to add it. In many cases it would mean to introduce another ng-container. Isn't it even worse?

Reply:
Maybe it will be less readable, but for example `permissions` observable does some computations and thus the calculation is done more than once (once or each pipe).

Reply:
All of these are used just once...